reference/content-types/smart_content.rst

```diff
@@ -0,0 +1,71 @@
+Smart content
+=============
+
+Description
+-----------
+
+Shows a list of pages, which depends on a configureable filter. You can define
+which which site's children should be included, what tags the filtered pages
+must have, how they are sorted, and how many results you want to get.
+Additionally you can define some presentation types, so that the content 
+manager can decide if the pages should be displayed e.g. in one column or two
+columns. The filter is saved as a JSON string in the database.
+
+Parameters
+----------
+
+.. list-table::
+    :header-rows: 1
+
+    * - Parameter
+      - Type
+      - Description
+    * - max_per_page
+      - integer
+      - Limits the results per page. This parameter also enables the paging of
+        the results at all.
```

Comment:
`If this parameter is not specified, pagination will be disabled` or `Omit this parameter to disable pagination`